### ISSUE TITLE
Neater display for the date

### ISSUE THREAD

**Issue by ellyasafaa** [onaio/rdt-standard#810]
![klik tombol ini hingga berwarna hijau untuk memunculkan detail pertanyaan kontak (tracing kontak) (15)](https://user-images.githubusercontent.com/73918009/108791358-75be5080-75b1-11eb-90eb-6d2dc22832e2.png)

**Comment by vincent-karuri** [comment]
Ok, can be done in a future release.

**Comment by ellyasafaa** [comment]
@vincent-karuri Yes, please proceed.

**Comment by syedowaisali90** [comment]
@ellyasafaa in English the date duration label is `Test Date` and in Bahasa, it is something like `Tanggal pemeriksaan COVID-19` so could you confirm that is a correct translation in Bahasa.

cc: @mwkhan1983 @vincent-karuri

**Comment by mwkhan1983** [comment]
@ellyasafaa please need your input on this. See Owais comment above

**Comment by ellyasafaa** [comment]
@syedowaisali90 Yes, that is correct. However, after seeing the context, it is about the duration and not the date, so the correct one should be: 

**Hari Periksa: 6 hari lalu**

6d is not in Bahasa, it is in English

Apart from this date, there are several date duration label that have the same issues. Have you looked at those?

**Comment by syedowaisali90** [comment]
@ellyasafaa could you shorten the translation of `Test Date` in Bahasa.

**Comment by ellyasafaa** [comment]
> @ellyasafaa could you shorten the translation of `Test Date` in Bahasa.

As I mentioned, it would be best like this: 

**Test date: 6d** = **Hari Periksa: 6 hari lalu**

Can we make it like this?

**Comment by syedowaisali90** [comment]
yeah, that's fine

**Comment by syedowaisali90** [comment]
@ellyasafaa could you close this issue.
